packages/app/src/components/Titlebar.tsx

```
@@ -0,0 +1,42 @@
+  }, []);
+
+  return (
+    <div
+      className={`top-0 left-0 w-screen min-h-[31px] h-[31px] py-[5px] px-2.5 flex gap-2 bg-background-1 z-999 ${border && "border-b border-b-border"}`}
+      style={{ WebkitAppRegion: "drag" }}
+    >
+      {icon && (
+        <img
+          src={iconSvg}
+          className={`h-[15px] translate-y-[3px] ${focused ? "opacity-90" : "opacity-60"}`}
+        />
```

Biome (2.1.2)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)